src/Contract/Address.purs

```diff
@@ -141,8 +145,17 @@ getWalletAddress = head <$> getWalletAddresses
 -- | Get all the `Address`es of the browser wallet.
 getWalletAddresses
   :: forall (r :: Row Type). Contract r (Array Address)
-getWalletAddresses = do
-  addresses <- wrapContract QueryM.getWalletAddresses
+getWalletAddresses = getWalletAddressesGeneral Nothing
+
+-- | Get all the `Address`es of the browser wallet with pagination.
+getWalletAddressesPaginated
+  :: Paginate -> forall (r :: Row Type). Contract r (Array Address)
```

Comment:
Typically we lift forall's as high as possible

```suggestion
  :: forall (r :: Row Type). Paginate -> Contract r (Array Address)
```

Reply:
Thanks!